src/component/DesignerLink/DesignerLink.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export default function DesignerLink({designer, designerURL}) {
+  return (
+    <>
+      <Link href={designerURL} target="_blank" rel="noreferrer">
+        Designed by <span>{designer}</span>
+      </Link>
+    </>
```

Comment:
You don't necessarily need the fragments here as the `Link` is the sole container of your JSX.